index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that a <a>subject</a> controls, or a <a>holder</a>
+has been designated to use, one or more particular confirmation methods. This
+binds a specific <a>claim</a> about a <a>subject</a> to one or more
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+around how a <a>verifier</a> can validate that the <a>holder</a> is bound to
+the <a>claims</a> presented in a <a>verifiable credential</a>.
```

Comment:
```suggestion
A Confidence Method can be included by an <a>issuer</a> in a <a>verifiable
credential</a> to inform verifiers of mechanisms they could use to increase
their confidence that:

<ul>
a particular identifier in the <a>verifiable credential</a> refers to the
same <a>entity</a> the issuer intended it to refer to and, optionally,
</ul>

<ul>
the presenter of the <a>verifiable credential</a> is the same <a>entity</a>.
</ul>

A confidence method can be used by an <a>issuer</a> to, for example, declare
that an <a>entity</a> controls or has been designated to use, one or more
mechanisms for demonstrating proof-of-possession (or proof-of-use) of
cryptographic key material. Alternatively, an <a>issuer</a> could declare,
for example, that an <a>entity</a> identified in the <a>verifiable credential</a>
can be checked against a biometric. In this way, an <a>issuer</a> explicitly
gives guidance around how a <a>verifier</a> can increase their confidence
that an <a>entity</a>, such as the presenter of a <a>verifiable credential</a>,
is the same <a>entity</a> that the <a>issuer</a> made claims about.
```

Reply:
@dlongley — The `<ul>` is misused here. As written, the prose of these bullets should be part of the preceding paragraph (from which the current terminal colon should be removed).

Also, in the second bullet, it's unclear what `same` is comparing — that is, `the presenter of the <a>verifiable credential</a> is the same <a>entity</a>` *as which other entity? <a>subject</a>? <a>issuee</a>? other?*

After dealing with the above, it became clear that the second pair of bullets belonged in the same list as the first, and that the concluding paragraph was redundant against the list items.

I've incorporated the above comments into the following:

```suggestion
An <a>issuer</a> can include a Confidence Method in a <a>verifiable
credential</a> to inform verifiers of mechanisms they could use to increase
their confidence in the truth of a variety of things, including the 
following:</p>
<ul>
  <li>
a particular identifier in the <a>verifiable credential</a> refers to the
same <a>entity</a> the issuer intended it to refer to
  </li>
  <li>
an <a>entity</a>, such as the presenter of a <a>verifiable credential</a>, 
is the same <a>entity</a> that the <a>issuer</a> made claims about
  </li>
  <li>
an <a>entity</a> controls, or has been designated to use, one or more 
mechanisms for demonstrating proof-of-possession or proof-of-use of 
cryptographic key material
  </li>
  <li>
an <a>entity</a> identified in the <a>verifiable credential</a> can be 
checked against a biometric
  </li>
</ul>
```

Reply:
I'm happy with your suggestions here, @TallTed. I had also apparently forgotten how to use `ul` properly -- so thanks for that.